api/src/main/java/com/codeforcommunity/rest/subrouter/ProtectedSiteRouter.java

```diff
@@ -416,7 +417,13 @@ private void registerRejectSiteImage(Router router) {
   private void handleRejectSiteImage(RoutingContext ctx) {
     JWTData userData = ctx.get("jwt_data");
     int imageId = RestFunctions.getRequestParameterAsInt(ctx.request(), "image_id");
-    String rejectionReason = RestFunctions.getRequestParameterAsString(ctx.request(), "reason");
+
+    String rejectionReason;
+    try {
+      rejectionReason = RestFunctions.getRequestParameterAsString(ctx.request(), "reason");
+    } catch (MissingParameterException e) {
+      rejectionReason = "Your image upload was rejected by an admin";
+    }
```

Comment:
⛏️ Instead of doing a try/catch, can we instead use `RestFunctions.getoptionalQueryParam()` and work with an `Optional` instead? It's generally better to avoid using exceptions as control flow if we can.